src/components/PageIndex/FirstScreen/index.tsx

```diff
@@ -83,7 +19,23 @@ const FirstScreen: FC = () => {
   return (
     <section className={classnames([styles.firstScreen])}>
       <div className={styles.bg}></div>
-      <PWSellCountdown />
+
+      <div className={styles.bannerWrapper}>
+        <div className={styles.banner}>
+          <img src="/images/logo-builders-program.png" alt="Builders Program" />
+          <section className={styles.bannerContainer}>
+            <div className={styles.bannerBody}>
+              <header className={styles.bannerHeader}>
+                <h3 className={styles.bannerSubtitle}>We are thrilled to announce the launch of</h3>
+                <h2 className={styles.bannerTitle}>the Phala Builders Program!</h2>
+              </header>
+              <div className={styles.bannerText}>This program provides funding, technical guidance, and marketing support to innovative projects seeking to build on Phala’s decentralized, trustless compute cloud using <span className={styles.tag}>#Phat Contract</span>.</div>
```

Comment:
The `#Phat Contract` looks odd in frontend. Do we need the hashtag? Or should it be `#PhatContract`?

Reply:
Cool, updated.